Getting started with HtmlEditor 21 1 (#2242)
DevExpress/devextreme-documentation: concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/35 Configure Mentions.md

> [Mentions](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mentions/) allow users to reference others within text or conversation threads. This functionality is not used in this tutorial. To learn more about it, refer to the following demo: [Mentions](https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/Mentions/).

Reviewer: We gave literally no information about the feature. Maybe rephrase the sentence a bit?
```suggestion
[Mentions](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mentions/) allow users to reference others within text or conversation threads. This functionality is not used in this tutorial. To see this feature in action, refer to the following demo: [Mentions](https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/Mentions/).
```

Reviewer 2: ```suggestion
[Mentions](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/mentions/) allow users to reference others within text or conversation threads. This functionality is not used in this tutorial. For information about it, refer to the following demo: [Mentions](https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/Mentions/).
```
